SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

fopen-safer.c for whatever reason does not exist when added by gnulib.

--- lib/Makefile.am	2021-04-11 19:01:01.265993928 +1000
+++ lib/Makefile.am	2021-04-11 19:01:40.413232722 +1000
@@ -31,7 +31,8 @@
 lib_SOURCES = \
   get-errno.h get-errno.c \
   subpipe.h subpipe.c \
-  $(bitsets_sources) $(additional_bitsets_sources) $(timevars_sources)
+  $(bitsets_sources) $(additional_bitsets_sources) $(timevars_sources) \
+  fopen-safer.c
 
 # Implementation of bitsets
 bitsets_sources = \
